root.jsx

```diff
@@ -25,18 +26,22 @@ function preRenderSetup(callwhendone) {
         if (msg === 'ResizeObserver loop limit exceeded') {
             return;
         }
-        var l = {};
-        l.level = 'ERROR';
-        l.message = 'msg: ' + msg + ' row: ' + line + ' col: ' + column + ' stack: ' + stack + ' url: ' + url;
-
-        const req = new XMLHttpRequest();
-        req.open('POST', '/api/v4/logs');
-        req.setRequestHeader('Content-Type', 'application/json');
-        req.send(JSON.stringify(l));
 
+        let displayable = false;
         if (isDevMode()) {
-            store.dispatch(logError({type: 'developer', message: 'DEVELOPER MODE: A JavaScript error has occurred.  Please use the JavaScript console to capture and report the error (row: ' + line + ' col: ' + column + ').'}, true));
+            displayable = true;
         }
+
+        store.dispatch(
+            logError({
+                type: AnnouncementBarTypes.DEVELOPER,
+                message: 'A JavaScript error has occurred. Please use the JavaScript console to capture and report the error (msg: ' + msg + ', row: ' + line + ', col: ' + column + ').',
```

Comment:
We could probably file this as a separate ticket, but maybe we could send this error to the server without the UI text? It seems weird for the logs on the server to include the "Please use the JS console..." part

Reply:
Sure, please see cb9e074f1360c6e4fc54f970c29cf23fe7dca051.